packages/compass-components/src/components/resize-handle.tsx

```diff
@@ -0,0 +1,89 @@
+/** @jsx jsx */
+import { css, jsx } from '@emotion/react';
+
+const LEFT_ARROW_KEY = 'ArrowLeft';
+const RIGHT_ARROW_KEY = 'ArrowRight';
+const UP_ARROW_KEY = 'ArrowUp';
+const DOWN_ARROW_KEY = 'ArrowDown';
+
+const baseResizerStyles = css({
+  position: 'absolute',
+  background: 'transparent',
+  left: 0,
+  bottom: 0,
+  right: 0,
+  top: 0,
+  padding: 0,
+  transition: 'opacity 250ms ease',
+  transitionDelay: '0ms',
+  backgroundColor: '#019EE2',
+  opacity: '0',
+  outline: 'none',
+  ':focus': {
+    opacity: 1,
+  },
+});
+
+const verticalResizerStyle = css({
+  left: '3px',
+  right: '3px',
+});
+
+const horizontalResizerStyle = css({
+  top: '4px',
+  bottom: '2px',
+});
+
+function ResizeHandleVertical({
+  onMoveRightPressed,
+  onMoveLeftPressed,
+}: {
+  onMoveRightPressed: () => void;
+  onMoveLeftPressed: () => void;
+}): React.ReactElement {
+  return (
+    // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions
+    <div
+      css={[baseResizerStyles, verticalResizerStyle]}
+      onKeyDown={(e) => {
+        if (e.key === LEFT_ARROW_KEY) {
+          onMoveLeftPressed();
+        } else if (e.key === RIGHT_ARROW_KEY) {
+          onMoveRightPressed();
+        }
+      }}
+      // eslint-disable-next-line jsx-a11y/no-noninteractive-tabindex
+      tabIndex={0}
```

Comment:
> I was investigating using an input html element for more accessibility, but I couldn't find an easy way for the width to be dynamically sized based on the user's viewport for the range element. We can revisit down the line if it becomes higher priority I think.

Can you explain a bit more what's the issue of using range element here? I would really don't want us to start with `compass-components` by disabling the a11y rules. Maybe we can discuss it and try to find an approach that works?

If there is no way we can follow Slacks implementation and use a range input, we should consider providing this information with better set of aria attributes (I am not sure that `separator` is a good match here, it's also non-interactive, this is probably why jsx-a11y is still not happy with this code even though you added a role), here's a [slider role description](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_slider_role) for example that might fit our use-case better